app/helpers/users_helper.rb

```diff
@@ -0,0 +1,27 @@
+module UsersHelper
+  def positions_list
+    Position.pluck(:name, :id)
+  end
+
+  def departments_list
+    arr = Department.pluck(:name, :id)
+    # arr << ["", nil]
+    arr
+  end
+
+  def user_department(user)
```

Comment:
In general, it will be better to move user related methods to the User model. Then you will be able to call them everywhere

Reply:
Also, you can just use `user.department&.id`. It will do the same

Reply:
DelI removed the methods: user_position, user_department, from the user_helper.rb,  they are not needed.

Reply:
Branch WM6